internal/services/containers/kubernetes_flux_configuration_resource_test.go

```diff
@@ -231,6 +231,41 @@ func TestAccKubernetesFluxConfiguration_kustomizationPostBuildUpdate(t *testing.
 	})
 }
 
+func TestAccKubernetesFluxConfiguration_gitRepositoryProvider(t *testing.T) {
```

Comment:
Shouldn't we break this up into two separate test cases? (e.g.,  `TestAccKubernetesFluxConfiguration_gitRepositoryProviderBasic` and `TestAccKubernetesFluxConfiguration_gitRepositoryProviderUpdate`)?
```suggestion
func TestAccKubernetesFluxConfiguration_gitRepositoryProviderBasic(t *testing.T) {
```